packages/console/src/components/Navigation/DefaultAppBarContent.tsx

```diff
@@ -74,27 +109,103 @@ export const DefaultAppBarContent = (props: DefaultAppBarProps) => {
     },
   ];
 
+  const styles = makeStyles(() => ({
+    wordmark: {
+      position: 'relative',
+      paddingTop: '22px',
+      '& > svg': {
+        height: '22px',
+        transform: 'translateX(-34px)',
+        marginTop: '4px',
+        top: '0',
+        position: 'absolute',
+      },
+      '& > svg > path:first-child': {
+        display: 'none',
+      },
+    },
+    flex: {
+      display: 'flex',
+    },
+  }))();
+
   return (
-    <>
-      <Link
-        className={classnames(commonStyles.linkUnstyled)}
-        to={Routes.SelectProject.path}
-      >
-        <FlyteLogo size={32} />
-      </Link>
-      {props.items?.length > 0 ? (
-        <NavigationDropdown items={props.items} console={props.console} />
-      ) : (
-        false
-      )}
-      <div className={styles.spacer} />
-      {isFlagEnabled && <OnlyMine />}
-      <UserInformation />
-      <AppInfo
-        versions={versions}
-        documentationUrl="https://docs.flyte.org/en/latest/"
-      />
-    </>
+    <Grid
+      container
+      direction={isLayoutHorizontal ? 'column' : 'row'}
+      justifyContent="space-between"
+      alignItems="center"
+      style={{ width: '100%', height: '100%' }}
+    >
+      <Grid item>
+        <Grid
+          container
+          direction={isLayoutHorizontal ? 'column-reverse' : 'row'}
+          alignItems="center"
+          spacing={2}
+        >
+          {isMobileNav && (
+            <Grid item className={styles.flex}>
+              <IconButton
+                color="inherit"
+                aria-label="open drawer"
+                onClick={handleSideNavToggle}
+              >
+                <MenuIcon>menu</MenuIcon>
+              </IconButton>
+            </Grid>
+          )}
+          <Grid item className={styles.flex}>
+            <Link
+              className={
+                isLayoutHorizontal
+                  ? commonStyles.linkUnstyled
+                  : classnames(commonStyles.linkUnstyled, styles.flex)
+              }
+              to={Routes.SelectProject.path}
+            >
+              <FlyteLogo size={32} hideText={isLayoutHorizontal} />
+              {isLayoutHorizontal && (
+                <Box className={styles.wordmark}>
+                  <FlyteLogo size={32} />
+                </Box>
+              )}
+            </Link>
+            {props.items?.length > 0 ? (
```

Comment:
We should return null and not false. Also this can be made shorter using && instead of ?: